pages/learn/welcome/security.md

```diff
@@ -50,7 +50,7 @@ Both the Docker pull request and the actual image download process are performed
 
 When the VPN is enabled, SSH access is available to the service using the {{ $names.company.lower }} dashboard or the CLI.
 
-The {{ $names.company.lower }} VPN disallows device-to-device traffic and prohibits outbound traffic to the Internet.  If a device were compromised, this ensures that it cannot contaminate another device.
+The {{ $names.company.lower }} VPN disallows device-to-device traffic and prohibits outbound traffic to the Internet.  If a device were compromised, this ensures that it cannot contaminate another device. To achieve this the VPN service is configured to run with iptables default `FORWARD` policy set to `DROP` and we do not set OpenVPN `--client-to-client` config option server side, so there is no physical way for the traffic between clients to traverse the interface(s). 
```

Comment:
```suggestion
The {{ $names.company.lower }} VPN disallows device-to-device traffic and prohibits outbound traffic to the Internet.  If a device were compromised, this ensures that it cannot contaminate another device. To achieve this the VPN service is configured to run with iptables default `FORWARD` policy set to `DROP` and we do not set OpenVPN `--client-to-client` config option server side, so there is no way for the traffic between clients to traverse the interface(s). 
```
I hesitate to use the word "physical" in this context, as they are still physically on the same cluster.

Reply:
Sounds sensible

Reply:
Is there some docs we can link here to show what option this is?